overlays/nixpkgs.nix

```diff
@@ -88,4 +88,22 @@ final: prev:
       };
     }
   );
+
+  # The fragment checks in 0.19.1 are broken.
+  # ToDO(katexochen): Check on lychee versions >0.19.1.
```

Comment:
I believe they are actually still being introduced :)

Fragments from the `id` attr work in latest releases, but from the `name` attr don't (https://github.com/lycheeverse/lychee/issues/1790)

Fragments were just unchecked (in full URLs?) in versions before v0.19.0.